blockchain-agent-proof/pom.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>com.sphereon.alfresco.blockchain-agent</groupId>
+        <artifactId>blockchain-agent-parent</artifactId>
+        <version>0.0.2-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>blockchain-agent-proof</artifactId>
+
+    <packaging>jar</packaging>
+
+    <properties>
+        <blockchain-proof-sdk-java8.version>0.10.0</blockchain-proof-sdk-java8.version>
+        <easy-blockchain-lib.version>0.1.3</easy-blockchain-lib.version>
+        <crypto-keys-sdk-java8-okhttp-gson.version>0.9.0</crypto-keys-sdk-java8-okhttp-gson.version>
+        <authentication-lib.version>0.1.4-SNAPSHOT</authentication-lib.version>
+        <sphereon-commons-bom.version>1.0.4-SNAPSHOT</sphereon-commons-bom.version>
```

Comment:
I think this also pulls in too many opinionated/old versions

Reply:
True. For now I decided to at least separate Proof API code and Factom code. They will both still have some dependencies on Sphereon Open source libraries which we can remove in a later PR in the interest of time.